src/Events/VideoJumped/index.js

```diff
@@ -0,0 +1,16 @@
+import { SOCKET_EVENTS } from '../../Constants';
+import RoomService from '../../Services/Room';
+
+export const jumpInVideo = async (
+  { id, video, duration }, { socket, io }
+) => {
+  await RoomService.jumpInVideo(id, duration);
+  io.to(id).emit(SOCKET_EVENTS.VIDEO_JUMPED, { video });
```

Comment:
burada bu fonksiyona parametre olarak gelen `video` değeri değil redise'deki güncel video geri dönülmelidir.

Örnek olarak benim fe'deki `video` state'imin `duration` değeri 15 olsun, arayüzde 45.saniyeye bastığım anda olay tetiklendi ve değeri 45 olan video buraya gönderildi. Ben redisteki `video`  `duration` değerini 45 yaptım ve ardından parametreden gelen videoyu döndüğüm için herkese değeri `15` olan videoyu döndüm bundan dolayı da hiçbir şey değişmedi.

Dışarıdan sadece oda id'si ve atlanacak saniye alınmalı geri kalan işi ben içeride yapıp kullanıcıya dönmeliyim.